keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,321 @@
+  - [Schedule Policy](#schedule-policy)
+  - [Feedback survey](#feedback-survey)
+- [Implementation History](#implementation-history)
+  - [Leads meeting feedback session](#leads-meeting-feedback-session)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input (including test refactors)
+- [ ] (R) Graduation criteria is in place
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+With this KEP, SIG Release proposes to change the current Kubernetes release
+cadence from 4 down to 3 releases per year.
+
+## Motivation
+
+Discussions around changing the release cadence for Kubernetes, which currently
+releases 4 times per year, are ongoing in the community.
+
+The extended release schedule for 1.19 resulted in only three minor Kubernetes
+releases for 2020. As a result, SIG Release received several questions across a
+variety of platforms and communication channels about whether the project
+intends to only have three minor releases/year.
+
+### Goals
+
+#### Enhance determinism
+
+With the current release cadence we already achieve a deterministic schedule
+for every year. The goal of this KEP is to increase this even further by
+providing a lightweight policy around creating the release schedule. Going
+down to 3 releases provides additional room for triage, development, and
+explicit breaks, which should result in better overall planning and more
+predictability.
+
+#### Reduce risk
+
+With higher predictability we can reduce the overall risk of changing the
+release schedule. The planning overhead of SIG Release gets reduced, while
+users of Kubernetes gain more time to adapt to the latest release.
+
+The current Kubernetes release cadence is so fast that most organizations
+cannot keep up with regularly making a minor version update every 3 months or
+going out of security support in a year. While releasing more frequently
+theoretically reduces the churn and risk of each release, this is only true if
+end users are actually able to apply the upgrades.
```

Comment:
yeah, the upgrade history is something I think we should really focus, but that is material for another KEP

Reply:
Thank you for the input! We should follow-up with the other SIGs after this KEP has been merged. :+1: